__tests__/html/useFocus.main.html

```diff
@@ -31,7 +31,7 @@
         await pageConditions.uiConnected();
 
         // WHEN: The callback of "useFocus()" is called.
-        await renderWebChatWithHook(() => window.WebChat.hooks.useFocus()());
+        await renderWebChatWithHook(() =>  Promise.resolve(window.WebChat.hooks.useFocus()).then(focus => focus()));
```

Comment:
Why we need to wait for useFocus()?

Reply:
As we're using `use-propagate`, we can't call focus synchronously in render anymore. Otherwise, we'll get the warning:
https://github.com/compulim/use-propagate/blob/main/packages/use-propagate/src/createPropagation.tsx#L36

```
          'use-propagate: The propagate callback function should not be called while rendering, ignoring the call.'

```

Reply:
Oh yea... you are using `Promise.resolve()` for "next idle callback". 😆